index.js

```diff
@@ -102,3 +102,27 @@ io.on('connection', socket => {
         console.log(`${currentUser} score: ${score}`);
     });
 });
+
+//Functions to perform fullscreen and chat toggle
+function goFullScreen() {
```

Comment:
Please do not add frontend code to this file. This file is primarily for the socket communication. See the files in /src/